libs/blocks/video/video.js

```diff
@@ -8,6 +8,8 @@ export default function init(a) {
   const video = `<video ${attrs}>
         <source src=".${pathname}" type="video/mp4" />
       </video>`;
-  a.insertAdjacentHTML('afterend', video);
-  a.remove();
+  if (a.parentNode) {
+    a.insertAdjacentHTML('afterend', video);
+    a.remove();
+  }
```

Comment:
Added check for a.parentNode as if background video is an autoblock and in async block loading, if block processes the background first, the video autoblock would fail. This would throw a console error.